task3/update/styles.css

```diff
@@ -0,0 +1,159 @@
+header{
```

Comment:
The css have a lot of technics to layout the content:
- the basic blocks (normal flow)
- the text flow (display: inline)
- the flexbox 
 mixed variation
position absolute/fixed is an additional ability that should be used with the purpose
the current approach approach will not give you anything good, the site will not be responsive.